.ado/windows-vs-pr.yml

```
@@ -546,3 +546,11 @@ jobs:
       #    script: npm test
       #    workingDirectory: current
       #  condition: and(succeeded(), or(eq(variables['BuildConfiguration'], 'DebugBundle'), eq(variables['BuildConfiguration'], 'ReleaseBundle')))
+
+  - template: templates/e2e-test-job.yml  # Template reference
```

```suggestion
  - template: templates/e2e-test-job.yml
```
Comment doesn't seem helpful